bin/api_diff.ml

```diff
@@ -1,21 +1,13 @@
-let _ = Api_watch_diff.diff_interface ~reference:"reference" ~current:"current"
-
 let run (`Ref_cmi reference) (`Current_cmi current) =
   let current = Cmi_format.read_cmi current in
   let reference = Cmi_format.read_cmi reference in
-  let typing_env = Env.empty in
-  let coercion1 () =
-    Includemod.signatures typing_env ~mark:Mark_both reference.cmi_sign
-      current.cmi_sign
-  in
-  let coercion2 () =
-    Includemod.signatures typing_env ~mark:Mark_both current.cmi_sign
-      reference.cmi_sign
+  let coercion =
```

Comment:
`coercion` is a bit of cryptic term here and we were using it before because it helped understand how we used `Includemod`. Now that the implementation has been moved to the library, the fact that we use `Includemod` becomes an implementation detail we don't have to know about here anymore.

I suggest using a more meaningful word here such as `differs`, as in "does current differs from reference?".

Reply:
> I suggest using a more meaningful word here such as `differs`, as in "does current differs from reference?".

Sorry, since the logic has been encapsulated in the library, I should have used a more meaningful name. I will rectify it asap. Thanks for pointing it out @NathanReb!